example/js/example_settings.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+class ExampleSettings extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onBlurSelection = this.onBlurSelection.bind(this);
```

Comment:
I was worried about class properties not being in the spec yet, but that really is much easier...  I just got it working using babel-eslint so I think I'll change all of these over.